javascript/javascript2/week2/home.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Quiz App</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">    
+    <link rel="stylesheet" href="home.css">
+
+    <title>Document</title>
```

Comment:
You have included two `<title>` tags for your HTML document. I think you meant to get rid of this one.